CHANGELOG.md

```diff
@@ -1,5 +1,27 @@
 # Change Log
 
+# [1.26.0 (July 27th, 2023)](https://github.com/eclipse/eclipse.jdt.ls/milestone/116?closed=1)
+ * performance - Check completion proposal is compatible or not. See [#2733](https://github.com/eclipse/eclipse.jdt.ls/pull/2733).
+ * enhancement - Add support for smart semicolon insertion. See [#2710](https://github.com/eclipse/eclipse.jdt.ls/pull/2710).
+ * enhancement - Add code actions for Join/Split variable. See [#2732](https://github.com/eclipse/eclipse.jdt.ls/pull/2732).
+ * enhancement - Support outline view for decompiled source. See [#2742](https://github.com/eclipse/eclipse.jdt.ls/pull/2742).
+ * enhancement - Send a `SourceInvalidatedEvent` to client when source attachment is updated for jar libraries. See [#2764](https://github.com/eclipse/eclipse.jdt.ls/pull/2764).
+ * enhancement - Log error when Gradle and JDK versions are mismatched. See [#2749](https://github.com/eclipse/eclipse.jdt.ls/pull/2749).
+ * bug fix - Fix parameter and exception changes in method signature refactoring. See [#2754](https://github.com/eclipse/eclipse.jdt.ls/pull/2754).
+ * bug fix - Fix incorrect semantic highlighting due to out-of-date AST in use. See [#2709](https://github.com/eclipse/eclipse.jdt.ls/pull/2709).
+ * bug fix - Prevent caching outdated AST in `CoreASTProvider`. See [#2714](https://github.com/eclipse/eclipse.jdt.ls/pull/2714).
+ * bug fix - Call Hierarchy resolves wrong element under certain conditions. See [#2771](https://github.com/eclipse/eclipse.jdt.ls/issues/2771).
+ * bug fix - Do not delete the Java project nature removing a nested `.classpath` resource file. See [#2750](https://github.com/eclipse/eclipse.jdt.ls/pull/2750).
+ * bug fix - "Unable to acquire the state change lock for the module 'buildship'". See [#2758](https://github.com/eclipse/eclipse.jdt.ls/pull/2758).
```

Comment:
typos. Unnecessary quotes `""`

Reply:
I've removed them. I wanted to quote it since it was technically a log message but it's not that cryptic to leave as is.